src/components/Artist.jsx

```diff
@@ -0,0 +1,37 @@
+
+export const Artist = ({ artistInfo }) => {
+
+	const isMultipleArtists = artistInfo.artists.length > 1
+
+	// for multiple artists in an album or single
+	let artistName
+	if (isMultipleArtists) {
+		artistName = artistInfo.artists.map(artist => artist.name || 'Unknown').join(', ')
```

Comment:
It's easy to understand the Artist component and the handling of artist information. Your approach to handling multiple artists and ensuring link security was clever.

Reply:
Thanks  it was a shared effort :)